draft-jhoyla-tls-extended-key-schedule.md

```diff
@@ -174,6 +179,24 @@ heading. It should be administered under a Specification Required policy {{!RFC8
 
 --- back
 
+# Sample Potential Use Cases
+
+This structure can be used to implement a number of different effects naturally.
+The draft provides an importer interface that mirrors the exporter interface.
+This provides a way to layer TLS on to other protocols, as opposed to layering
+other protocols over TLS. This can be used for hybrid key exchanges, which, in
+effect, is layering TLS over a secondary AKE. Although the key exchanges are
+interleaved, the post-quantum AKE completes first, as demonstrated by its output
+key being used as an input for computing TLS's master secret.
```

Comment:
This is a bit misleading. The exporter interface is, well, an actual interface. This document does not provide an *explicit* importer interface, but a way for other drafts to add those interfaces. I'd suggest:

```suggestion
The draft provides a mechanism for importing additional information into the TLS key schedule.
Future applications and specifications can use this mechanism to layer TLS on to other protocols, 
as opposed to layering other protocols over TLS. For example, as discussed in {{intro}}, this can 
be used for hybrid key exchange, which, in effect, is layering TLS over a secondary AKE. 
Although the key exchanges are interleaved, the post-quantum AKE completes first, as demonstrated 
by its output key being used as an input for computing TLS's master secret.
```